internal/constellation/helm/generateCertManager.sh

```diff
@@ -18,4 +18,21 @@ helm pull cert-manager \
   --untar \
   --untardir "charts"
 
+echo "Pinning cert-manager images..."
+# Manually copy the sha256 digest from the corresponding repository:
+# quay.io/jetstack/cert-manager-controller
+yq -i '(.image.digest) = "sha256:fb2546fe51e49206dbf72bb0d6f909a0018eda0c2b024547b03d3f3d604e4c5e"' charts/cert-manager/values.yaml
```

Comment:
We could auto-generate the hashes along the lines of

```bash
curl -L -H "Accept: application/vnd.docker.distribution.manifest.v2+json" https://quay.io/v2/jetstack/${component}/manifests/v${version} | sha256sum
```

Reply:
thanks, implemented that now